cdm/src/main/java/ucar/nc2/ft2/coverage/adapter/SimpleGeometryCSBuilder.java

```diff
@@ -99,22 +108,24 @@ private FeatureType classify () {
     	return FeatureType.SIMPLE_GEOMETRY;
     }
 
-    // what makes it a grid?
-    // each dimension must have its own coordinate variable
-    Set<Dimension> indDimensions = CoordinateSystem.makeDomain(independentAxes);
-    Set<Dimension> allDimensions = CoordinateSystem.makeDomain(allAxes);
-    if (indDimensions.size() == allDimensions.size()) {
-      return FeatureType.GRID;
-    }
-
-    // default
-    return FeatureType.COVERAGE;
+    // Default, station is similar to Simple Geometry
+    return FeatureType.STATION;
```

Comment:
Why is it preferable to return `STATION` instead of returning null, logging an error/warning, or throwing an Exception?

Reply:
Perhaps that may be changed to null but it may lead to some interesting behavior when somethings try to operate on that. But null is probably more fitting as you suggest.

Station was the default value perhaps for compatibility reasons, but that may not even apply in this case.